vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -403,8 +404,10 @@ func (c completedConfig) New(name string, delegationTarget DelegationTarget) (*G
 
 		listedPathProvider: apiServerHandler,
 
-		swaggerConfig: c.SwaggerConfig,
-		openAPIConfig: c.OpenAPIConfig,
+		swaggerConfig:        c.SwaggerConfig,
+		openAPIConfig:        c.OpenAPIConfig,
+		swaggerAPIContainers: append([]*restful.Container{apiServerHandler.GoRestfulContainer}, delegationTarget.SwaggerAPIContainers()...),
+		openapiDelegateSpec:  delegationTarget.OpenAPISpec,
```

Comment:
if you stored the `delegationTarget`, you would have to store all the `delegate` methods and containers. Would make it much easier to reason about.